docs/testing-components/index.md

```diff
@@ -3,5 +3,7 @@ title: Testing Your Components
 ---
 
 *   [Unit Testing](./unit-testing/index.md) - How to start Unit Testing
+*   [UI Testing](./ui-testing/index.md) - How to start Ui Testing
+*   [Screenshot Testing](./screenshot-testing/index.md) - How to start Screenshot Testing
```

Comment:
The original ticket intent was to add these to the `contributing` page.
I think we should provide more default setting to run tests in enact apps. 
I willing to do this, but we need the opinion of others.